app/views/About.js

```diff
@@ -53,7 +97,19 @@ class AboutScreen extends Component {
             {languages.t('label.commitment_para')}
           </Text>
 
-          <SvgXml style={styles.aboutSectionIconTeam} xml={team} />
+          <SvgXml
+            onPress={this.handleTapTeam.bind(this)}
```

Comment:
it's already this bound because it used `handleTapTeam = () => {}` you can remove .bind(this)

Reply:
I based this on the code above for this.backToMain.  But dropping to just
```javascript
onPress={this.handleTapTeam}
```
Seems to work.